### DISCUSSION TITLE
Feature comparison from BigPicture.js

### DISCUSSION THREAD

**Discussion by cssyphus** [henrygd/bigger-picture#34] [General]
I just implemented BigPicture.js and quite like it, so am keen to play with BiggerPicture. What are the basic feature differences, in a nutshell?

Perhaps could you update your ReadMe to note the essential functional/implementation improvements? That is, what are the advantages to the Dev to upgrade to BiggerPicture.js from BigPicture.js?

I realize that not all people will be coming from BigPicture.js, but for those of us who are, it would be nice to have a "big picture overview" of the Big Picture updates, so to speak.

Loving it,
Thanks!

**Comment by henrygd** [comment]
Thanks for using the library.

Unfortunately the implementation for bigger-picture is completely different, You'll have to remove the bigpicture code and follow the readme -- or copy from the examples -- to implement bigger-picture. I'm not sure what I would put in a migration guide besides that.

As for a feature comparison, bigger-picture can do everything bigpicture does and includes extra features like image zooming, responsive images, inline galleries, html content display, and some accessibility improvements. The downside is that it's larger than bigpicture (but smaller than other libraries with the same features). bigpicture is fine if you just need something small and simple.

I'm going to hold off on adding a feature comparison to the readmes because I want to keep the focus pretty narrow on how to use them. Maybe I'll add a separate markdown file at some point and link to it, but I'm too busy to do it atm. Just got a dog.
